crypto/keyring/keyring.go

```diff
@@ -970,6 +974,20 @@ func (ks keystore) SetItem(item keyring.Item) error {
 	return ks.db.Set(item)
 }
 
+// bz, err := cdc.Marshal(privKey) yields an error that's why I cast privKey to curve PrivKey to serialize it
+func (ks keystore) NewLocalRecord(privKey types.PrivKey) (*Record_Local, error) {
+	fmt.Println("NewLocalRecord privKey", privKey)
+	fmt.Println("cdc", ks.cdc)
```

Comment:
@amaurym Although I have put `NewLocalRecord` on `keystore` , `cdc` is still nil while running  (Test_runAddCmdBasic)[https://github.com/cosmos/cosmos-sdk/blob/kerying-migration/client/keys/add_test.go#L22]

Reply:
It's worth digging in the code to see where ks.cdc is set in the test

Reply:
Resolved!